### PR TITLE
Display additional information on settings error (ie original cause) config/settings.py

### PR DESCRIPTION
**Pull request by txels**
I've used the occasion to include other small fixes.

### CODE REVIEW

File: deploystream/apps/oauth/github.py [hunk 1]
```diff
@@ -2,11 +2,12 @@
 from flask_oauth import OAuth
 
 from deploystream import app
-from deploystream.apps import oauth
+from deploystream.apps.oauth import get_token, set_token
```
Comment (txels) [inline deploystream/apps/oauth/github.py:5]: Turns out you were overriding `oauth` below. So I went back to your original imports.
No tests detected this.
We're a bit poor on coverage.

Comment (alexcouper) [inline deploystream/apps/oauth/github.py:5]: Ouch!

File: config/local_settings_carles.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 GITHUB_CONFIG = {
+    'organization': None,
```
Comment (alexcouper) [inline config/local_settings_carles.py:2]: Hmm - didn't think you needed this, think there's a default in the provider of None in which case it looks to your user's repos.

In your case though, shouldn't this be "pretenders"?

File: deploystream/apps/feature/lib.py [hunk 3]
```diff
@@ -11,7 +11,9 @@ def get_all_features(providers):
     all_features = []
 
     for provider in providers[IPlanningProvider]:
+        print("INFO: getting features from {0}".format(provider.name))
         for feature in provider.get_features():
+            print("INFO: found {0}".format(feature['title']))
```
Comment (alexcouper) [inline deploystream/apps/feature/lib.py:16]: Looking forward to the "Fix logging" story :)

File: tests/test_providers/test_sprintly_provider.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
+from mock import MagicMock, Mock, patch
+from nose.tools import assert_equal
+
+from deploystream.providers.sprintly import SprintlyProvider
+from supermutes.dot import dotify
+
+
+# Real data sample from sprint.ly, anonimised
```
Comment (alexcouper) [inline tests/test_providers/test_sprintly_provider.py:8]: I see what you mean about ideal pretenders stuff.

File: config/local_settings_carles.py (hunk 2 above)
Comment (txels) [inline config/local_settings_carles.py:2]: I use this config for personal projects. For pretenders, I use `CONFIG=alex` ;)

File: deploystream/apps/feature/lib.py (hunk 3 above)
Comment (txels) [inline deploystream/apps/feature/lib.py:16]: Yeah me too before this gets out of hand...

**Comment by alexcouper** [comment]
I'm online now if you want a chat @txels (Skype)

**Pull request merged**